samples/README.md

```diff
@@ -7,20 +7,4 @@ It's the samples of Java chassis
 4. code first sample
 5. customer handler sample
 6. extend metrics health checker sample
-7. write metrics data into separate text files sample
-
-## 1. Start the ServiceComb/Service Center
```

Comment:
I think starting Service-Center should be there in Readme, as our samples won't run without a working instance of Service-Center

Reply:
Yeah, we need to keep the Service-Center running instruction here.

Reply:
I just checked the README file from other examples, they have the service center start instruction. We should remove these lines.